test/e2e/flowaggregator_test.go

```diff
@@ -532,6 +505,105 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 			checkRecordsForFlows(t, data, podAIPs.ipv4.String(), svcC.Spec.ClusterIP, isServiceIPv6, false, true, false, false, checkBandwidth)
 		}
 	})
+
+	// Antctl tests ensure antctl is accessible in a Flow Aggregator Pod
```

Comment:
s/antctl is accessible/antctl is available

Reply:
updated